libsupport/include/katana/Strings.h

```diff
@@ -2,10 +2,16 @@
 #define KATANA_LIBSUPPORT_KATANA_STRINGS_H_
 
 #include <initializer_list>
+#include <sstream>
 #include <string>
 #include <string_view>
 #include <vector>
 
+#include <boost/archive/iterators/base64_from_binary.hpp>
+#include <boost/archive/iterators/binary_from_base64.hpp>
+#include <boost/archive/iterators/insert_linebreaks.hpp>
+#include <boost/archive/iterators/remove_whitespace.hpp>
+#include <boost/archive/iterators/transform_width.hpp>
```

Comment:
Would prefer these includes to be moved into the `.cpp` file. Callers don't need to know about any of these types.